### DISCUSSION TITLE
Pages Listed as Tabs

### DISCUSSION THREAD

**Discussion by markadrake** [umbraco/Umbraco.Forms.Issues#1513] [Ideas]
I have a complex form that has already reached six pages. There are logic jumps, and we are using both pages and groups to toggle the visibility of fields.

Working on the pages contextually would be great instead of having to scroll through one long page. One option would be to use tabs for each page. Whatever the method, it would be nice if it was optional and toggled. 

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/713355/ecbf7909-2bc6-4a5e-8efa-ace676cf8764)

**Comment by AndyButland** [comment]
Whilst not directly addressing the specific request, I'd just like to note we have added a "jump to page" dropdown in the latest 13.2 and 14.1 (out now in release candidates).  Screenshot and more details on the related issue [here](https://github.com/umbraco/Umbraco.Forms.Issues/issues/1243#issuecomment-2213886404).

**Comment by TvGessel** [comment]
Thank you for the suggestion. We have added this to our internal backlog for further discussion and prioritization to determine if and when we can include this in the roadmap.
